client/index.jsx

```diff
@@ -89,21 +114,97 @@ class App extends React.Component {
     })
   }
 
-  sort
+  onNextReviews() {
+    if(this.state.onProd === true) {
+      let counter = this.state.counter;
+      let productRevLength = this.state.prodReviews.length;
+      let maxPresses = Math.floor(productRevLength / 4);
+      if(counter <= maxPresses) {
+        let start = this.state.sIndex + 4;
+        let end = this.state.eIndex + 4;
+        let next = this.state.prodReviews.slice(start, end);
+        counter ++;
+        this.setState({
+          counter: counter,
+          sIndex: start,
+          eIndex: end,
+          displayed: next,
+        })
+      }
+    } else {
+      let counter = this.state.counter;
+      let shopRevLength = this.state.shopReviews.length;
+      let maxPresses = Math.floor(shopRevLength / 4);
+      if(counter <= maxPresses) {
+        let start = this.state.sIndex + 4;
+        let end = this.state.eIndex + 4;
+        let next = this.state.shopReviews.slice(start, end);
+        counter ++;
+        this.setState({
+          counter: counter,
+          sIndex: start,
+          eIndex: end,
+          displayed: next,
+        })
+      }
+    }
+  }
+
+  onPreviousReviews() {
+    if(this.state.onProd === true) {
```

Comment:
I'm curious about what you mean by 'onProd'?

Reply:
I have a toggle in state called  "onProd" that is true if the product list is displayed, and becomes false when the shop reviews are displayed. This way, I can get a hold of the right arrays to slice and set to display when toggling between shop reviews and product reviews, and pressing the buttons to move forward or backwards through pages